docs/install.md

```diff
@@ -0,0 +1,76 @@
+## Installation
+
+Following ways are supported to run Cerberus:
+
+- Standalone python program through Git or python package
+- Containerized version using either Podman or Docker as the runtime
+- Kubernetes or OpenShift deployment
+
+**NOTE**: Only OpenShift 4.x versions are supported.
```

Comment:
Perhaps instead of supported
"NOTE: Only Openshift 4.x versions have been tested."

Reply:
Some of the checks are very specific to OCP 4.x like cluster operators etc., so it might not work on OCP 3.x. Modified it to "NOTE: Only OpenShift 4.x versions are tested and supported". PTAL.

Reply:
I'm still not sold on using the term supported. It implies a level or responsibility I'm not sure we want. But otherwise all good

Reply:
@dry923 updated it to tested, PTAL.